toolkit/offchain/src/reserve/release.rs

```diff
@@ -392,7 +393,7 @@ mod tests {
 			&reserve_data(),
 			&previous_reserve_utxo(),
 			&reference_utxo(),
-			20,
+			5,
```

Comment:
This 20 meant 10 over the other 10 from `previous_reserve_utxo()`, to not be confused by two different 10s, I've changed this to 5.